<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>D2Coding</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>754</width>
      <height>624</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>학생정보</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>19</x>
            <y>21</y>
            <width>311</width>
            <height>170</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>학생번호 </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_id">
               <property name="inputMask">
                <string/>
               </property>
               <property name="maxLength">
                <number>32764</number>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>학생이름 </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_name">
               <property name="toolTip">
                <string>학생이름을 입력하세요</string>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="placeholderText">
                <string>학생이름입력</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>핸드폰   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_mobile">
               <property name="toolTip">
                <string>핸드폰번호를 입력하세요</string>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="placeholderText">
                <string>핸드폰번호입력</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>입학년도 </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_regyear">
               <property name="toolTip">
                <string>입학년도를 입력하세요</string>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="placeholderText">
                <string>입학년도입력</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_7">
          <property name="geometry">
           <rect>
            <x>19</x>
            <y>199</y>
            <width>411</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="buttonArea">
           <item>
            <widget class="QPushButton" name="btn_add">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>추가</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_mod">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>수정</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_del">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>삭제</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="tblStudent">
         <property name="minimumSize">
          <size>
           <width>750</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>320</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
